### PR TITLE
Creating Gatsby x Tina starter

### PR DESCRIPTION
**Pull request by Freego1783**
D9493A89-0115-4E10-B435-67F1B6CABED4

Taken from my repo: https://github.com/Freego1783/tina-gatsby-test

Redid all steps in clean and separate commits.

### CODE REVIEW

File: gatsby-node.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import express, { Express } from "express";
+import type { GatsbyNode } from 'gatsby';
+import client from "./tina/__generated__/client";
+import path from "path";
+
+export const createPages: GatsbyNode['createPages'] = async ({ actions }) => {
+
+  const { createPage } = actions
+
+  const postsResult = await client.queries.postConnection({ first: 1000 });
```
Comment (JackDevAU) [inline gatsby-node.ts]: Do we need to specify the first 1,000 ? 

Can we leave this blank?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "private": true,
+  "description": "tina-gatsby-starter",
+  "author": "Hugo Pernet",
+  "keywords": [
+    "gatsby"
+  ],
+  "scripts": {
+    "dev": "npx tinacms dev -c \"gatsby develop\"",
+    "start": "gatsby develop",
+    "build": "tinacms build --no-client-build-cache && gatsby build",
```
Comment (JackDevAU) [inline package.json]: `--no-client-build-cache` do we need this anymore?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  "name": "tina-gatsby-starter",
+  "version": "1.0.0",
+  "private": true,
+  "description": "tina-gatsby-starter",
+  "author": "Hugo Pernet",
+  "keywords": [
+    "gatsby"
+  ],
+  "scripts": {
+    "dev": "npx tinacms dev -c \"gatsby develop\"",
```
Comment (JackDevAU) [inline package.json]: Does this need to use `npx` or can we remove it

File: tina/config.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  },
+  // See docs on content modeling for more info on how to setup new content models: https://tina.io/docs/schema/
+  schema: {
+    collections: [
+      {
+        name: "post",
+        label: "Posts",
+        path: "content/posts",
+        ui: {
+          router: ({ document }: { document: { _sys: { template: string; filename: string } } }) => {
```
Comment (JackDevAU) [inline tina/config.ts]: Shouldn't need to type out document 😲

File: tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,102 @@
+{
```
Comment (JackDevAU) [inline tsconfig.json:1]: Can we remove the commented out config options

**Review by JackDevAU — CHANGES_REQUESTED** [review]
🚀 Awesome stuff - I've left a few comments (small)

File: gatsby-node.ts (hunk 1 above)
Comment (Freego1783) [reply gatsby-node.ts]: ✅Fixed

> Do we need to specify the first 1,000 ?

Nope! I supposed we don't need this for the amount of data in the starter 😬

File: package.json (hunk 2 above)
Comment (Freego1783) [reply package.json]: ✅Fixed - removed

I tried without it and it works.!

File: package.json (hunk 3 above)
Comment (Freego1783) [reply package.json]: ✅Fixed - removed _npx_

File: tina/config.ts (hunk 4 above)
Comment (Freego1783) [reply tina/config.ts]: ✅Fixed 

Looks like Copilot went overboard with this 😬

File: tsconfig.json (hunk 5 above)
Comment (Freego1783) [reply tsconfig.json:1]: ✅Done

File: README.md [hunk 6]
```diff
@@ -1 +1,71 @@
-# tina-gatsby-starter
+![D9493A89-0115-4E10-B435-67F1B6CABED4](https://github.com/user-attachments/assets/3876c2bb-9f3d-4456-ab51-dc7722ac8c08)
+
+This Gatsby starter is powered by [TinaCMS](https://app.tina.io) for you and your team to visually live edit the structured content of your website. ✨
```
Comment (JackDevAU) [inline README.md]:
```suggestion
This Gatsby starter is powered by [TinaCMS](tina.io) for you and your team to visually live edit the structured content of your website. ✨
```

File: README.md [hunk 7]
```diff
@@ -1 +1,71 @@
...
+
+The content is managed through Markdown and JSON files stored in your GitHub repository, and queried through Tina GraphQL API.
+
+## Requirements
+
+- Git, [Node.js Active LTS](https://nodejs.org/en/about/releases/), yarn installed for local development.
+- A [TinaCMS](https://app.tina.io) account for live editing.
```
Comment (JackDevAU) [inline README.md]:
```suggestion
- A [TinaCMS](tina.io) account for live editing.
```

File: src/pages/index.tsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    </Layout>
+  );
+};
+
+export default IndexPage;
+
+export const Head: HeadFC = () => <title>Home Page</title>;
+
+export async function getServerData() {
+  const postsResult = await client.queries.postConnection({ first: 1000 });
```
Comment (JackDevAU) [inline src/pages/index.tsx]:
```suggestion
  const postsResult = await client.queries.postConnection();
```

File: LICENSE [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (JackDevAU) [inline LICENSE]:
```suggestion
   Copyright 2024 SSW USA
```

**Review by JackDevAU — CHANGES_REQUESTED** [review]
some more small changes 🙏

**Review by JackDevAU — APPROVED** [review]
🚀

**Pull request merged**